components/org.wso2.carbon.identity.api.user.push.device.handler.v1/src/main/resources/push-auth.yaml

```diff
@@ -0,0 +1,780 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: "WSO2 Identity Server Push-Based Authentication Rest API "
+  description: |
+    This document specifies  **RESTfulAPIs** for WSO2 **Identity Server - Push-based Authentication**.
+    It is written with [openapi:3.0.0 ](http://swagger.io/).
+  contact:
+    name: "WSO2"
+    url: "http://wso2.com/products/identity-server/"
+    email: "iam-dev@wso2.org"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+
+servers:
+  - url: https://{host}/t/{tenant_context}/api/users/v1
+
+security:
+  - OAuth2: [ ]
+  - BasicAuth: [ ]
+paths:
+  /me/push-auth/discovery-data:
```

Comment:
The me apis shouldn't need any permissions or scopes. Try out with a new user that doesn't have any roles.

Reply:
Removed